_config.yml

```diff
@@ -73,7 +73,7 @@ sass:
   style: :expanded # You might prefer to minify using :compressed
 
 # Use the following plug-ins
-gems:
+plugins:
```

Comment:
I'm getting a warning on this:

    Deprecation: The 'plugins' configurationoption has been renamed to 'plugins_dir'. Please update your config file accordingly.

I guess it should be renamed.

Reply:
Thanks. I got the same warning, just suggesting gems -> plugins. That said, your suggestion seems to work on my setup. I'll do a fixup.